toolchains/python/python.bzl

```diff
@@ -200,3 +200,184 @@ def nixpkgs_python_configure(
 
     if register:
         native.register_toolchains("@{}//:toolchain".format(name))
+
+
+def _nixpkgs_python_repository_impl(repository_ctx):
```

Comment:
Hmm, maybe there's a better name for this. Perhaps `_nixpkgs_python_poetry_packages?

Reply:
It is a generic implementation. The fact that poetry/poetry2nix have been used to generate the set is not important for the rule. It's generic on purpose. I think the updated description will be enough to clarify this.

Reply:
Fair enough. Maybe `_nixpkgs_python_packages` then? With `nixpkgs_python_repository` it could also refer to a Python interpreter repository. Just trying to avoid the ambiguity.